.circleci/config.yml

```diff
@@ -82,6 +82,7 @@ jobs:
       - run:
           name: Run All Unit Tests
           command: yarn nx run-many -t test
+          no_output_timeout: 20m
```

Comment:
The default is 10 minutes, and it seems like we're just surpassing it with 10m 25s...

Reply:
Yeah, and we already see that the build took more than twelve minutes and was successful. So it works. 👍